app/views/public_body/view_email.html.erb

```diff
@@ -47,10 +47,10 @@
 <div id="stepwise_make_request_view_email" class="stepwise_make_request_view_email">
   <strong>
     <% if @public_body.eir_only? %>
-      <%= link_to _("Make a new EIR request"), new_request_to_body_path(:url_name => @public_body.url_name)%>
+      <%= link_to _("Make a new EIR request to "), new_request_to_body_path(:url_name => @public_body.url_name)%>
     <% else %>
-      <%= link_to _("Make a new FOI request"), new_request_to_body_path(:url_name => @public_body.url_name)%>
+      <%= link_to _("Make a new FOI request to "), new_request_to_body_path(:url_name => @public_body.url_name)%>
```

Comment:
The previous version was better, as it only introduced a new string. This will _change_ strings, but won't solve the problem of joining partial strings.